chia/server/node_discovery.py

```diff
@@ -414,7 +414,7 @@ async def _periodically_serialize(self, random: Random) -> None:
             serialize_interval = random.randint(15 * 60, 30 * 60)
             await asyncio.sleep(serialize_interval)
             async with self.address_manager.lock:
-                await AddressManagerStore.serialize(self.address_manager, self.peers_file_path)
+                await AddressManagerStore.serialize_bytes(self.address_manager, self.peers_file_path)
```

Comment:
if you separate the serialization to bytes from the async disk write, you can write to disk without holding the lock

Reply:
Done